### ISSUE TITLE
How to select the checkpoint?

### ISSUE THREAD

**Issue by Up5XWJ** [DeepGraphLearning/ULTRA#35]
Based on the code in pretrain.py and the configuration in pretrain_3g.yaml, the pretraining process generates 10 checkpoints (one after each epoch).
I'm wondering which checkpoint is the released ultra_3g.pth. Since there are multiple checkpoints generated during training. 
Additionally, could you please clarify if there was a specific selection strategy(e.g., best performance on a validation set, last epoch, etc.) used to determine which checkpoint to release?
Thank you for your time and clarification.
